python/mxnet/ndarray/sparse.py

```diff
@@ -41,7 +41,7 @@
 from ..base import NotSupportedForSparseNDArray
 from ..base import _LIB, numeric_types
 from ..base import c_array_buf, mx_real_t, integer_types
-from ..base import mx_uint, NDArrayHandle, check_call
+from ..base import mx_uint, NDArrayHandle, check_call, mx_int64
```

Comment:
I thought we had an effort removing all user-defined types such as mx_uint. Why do we introduce them again? Can we use a standard data type such as ctypes.int64.

Reply:
Wasn't aware about it. Addressed it.